index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <title>Document</title>
```

Comment:
- [ ] Good job here but I think that `Please add a title ....` is kind of polite but a code reviewer should suggest in a more polite and humble way for example : `I would suggest you to add the descriptive and unique title here ....`.
- [ ] Also if code reviewer suggest in a more clear and descriptive way and also describe the reason for asked changes in a descriptive way, that would have been better because the more clear and descriptive our suggestion would be, the easier for the student to understand. for example you can describe your reason like that : `because having a unique and descriptive title for your webpage meets the best accessibility requirements and best SEO standards. that way the screen readers will easily read the title of your webpage and it will have good performance in search engine results.`